pubsub/gcloud/aio/pubsub/subscriber_client.py

```diff
@@ -43,9 +43,28 @@ def __init__(self, *args: List[Any], **kwargs: Dict[str, Any]) -> None:
             """
             self._flow_control = _FlowControl(*args, **kwargs)
 
+        def __repr__(self) -> str:
+            return self._flow_control.__repr__()
+
         def __getitem__(self, index: int) -> int:
             return self._flow_control[index]
 
+        @property
```

Comment:
Alternative option: dynamically generate these with `__getattr__`. 
```python
>>> class Foo:
...     def __init__(self, x):
...           self._x = x
...     def __getattr__(self, y):
...          return self._x[y]
...
>>> Foo({'a':42}).a
42
```
(or in this case, the body of `__getattr__` would `return getattr(self._flow_control, y)`)

Reply:
I thought about doing this but (1) RIP type hints and (2) we are pinned to <2.0.0 so our interface shouldn't change in a breaking way. I'm happy doing it like `__getattr__` regardless though. Thoughts?

Reply:
I made the changes here to use `__getattr__` and added a unit test, but I hadn't realized that with this setup we get `pylint` errors on `FlowControl` in the form of `Instance of 'FlowControl' has no 'max_messages' member (no-member)` for each attribute. But with a toy example which _should_ be identical this error doesn't crop up. Okay this is very weird.

Reply:
Okay, this is just because `pylint` was only looking at the first definition of `FlowControl` which resides within `if BUILD_GCLOUD_REST:` which of course is unimplemented and has no `__getattr__` method.

Reply:
LMK if this looks reasonable!

Reply:
Yeah, I really like this approach -- delegating absolutely everything to the parent given this is a pass through class makes a lot of sense to me!